companies/tvinfo.json

```diff
@@ -0,0 +1,22 @@
+{
+    "slug": "tvinfo",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "entertainment"
+    ],
+    "name": "TVinfo Internet GmbH",
+    "address": "Kronprinzenstr. 49\n40217 Düsseldorf",
+    "phone": "+49 211 97262990",
+    "fax": "+49 211 97262999",
+    "email": "support@tvinfo.de",
+    "web": "https://www.tvinfo.de",
+    "sources": [
+        "https://www.tvinfo.de/impressum",
+        "https://www.tvinfo.de/datenschutz",
+        "https://github.com/datenanfragen/data/pull/1325"
+    ],
+    "suggested-transport-medium": "email",
```

Comment:
```suggestion
```

Reply:
`"suggested-transport-medium": "email",`

Recommended medium "Email" has been removed? What is the reason? Because no specific data protection contact is available?